lib/fog/google/requests/storage_json/put_bucket_acl.rb

```diff
@@ -0,0 +1,56 @@
+module Fog
+  module Google
+    class StorageJSON
+      class Mock
+        def put_bucket_acl(_bucket_name, _acl)
+          Fog::Mock.not_implemented
+        end
+      end
+
+      class Real
+        # Change access control list for an Google Storage bucket
+        def put_bucket_acl(bucket_name, acl)
+          #           data = <<-DATA
+          # <AccessControlList>
+          #   <Owner>
+          #     #{tag('ID', acl['Owner']['ID'])}
+          #   </Owner>
+          #   <Entries>
+          #     #{entries_list(acl['AccessControlList'])}
+          #   </Entries>
+          # </AccessControlList>
+          # DATA
+
+          #           request(:body     => data,
+          #                   :expects  => 200,
+          #                   :headers  => {},
+          #                   :host     => "#{bucket_name}.#{@host}",
+          #                   :method   => "PUT",
+          #                   :query    => { "acl" => nil })
+        end
+
+        private
+
+        def tag(name, value)
+          # "<#{name}>#{value}</#{name}>"
+        end
+
+        def scope_tag(scope)
```

Comment:
Unused method argument - `scope`. If it's necessary, use `_` or `_scope` as an argument name to indicate that it won't be used. You can also write as `scope_tag(*)` if you want the method to accept any arguments but don't care about them.